config.example.json

```diff
@@ -10,6 +10,7 @@
   "transaction_retry_after_secs": 3,
   "quic_proxy_addr": null,
   "use_grpc": false,
+  "activate_leader_schedule": false,
```

Comment:
Should be `calculate_leader_schedule_form_geyser`

Reply:
Done